src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -113,6 +115,32 @@ function isPreview() {
     return $(document).find('.navbar').text().indexOf('Preview') !== -1;
 }
 
+function isModeration() {
+    return $('#moderationHintButton').length !== 0;
+}
+
+function bindModerationHintButton() {
+    if (!isModeration()) {
+        return;
+    }
+    
+    var expandText = '[Show more]';
+    var closeText = '[Show less]';
+    $('#moderationHintButton').text(expandText);
```

Comment:
@JosephineKwa I am considering that if later the text is going to be changed, one could simply change the text in `js` file. Since the text `<a>` in are going to be changed between `[More]` and `[Less]`, by putting the text inside `js` file could be more concise.